Issue form overhaul (#1569)
astronomer/docs: new-feature-template.md

# Write your own new feature documentation

To have the docs team create documentation for your new feature: 

> 1. Fill out this template

Reviewer: ```suggestion
1. Complete this template.

```

> 2. Create a new page on [Internal drafts](https://www.notion.so/astronomerio/Internal-Drafts-e09f6e1e64464f1cb4b3468a549bb45f)

Reviewer: ```suggestion
2. Add a new page to [Internal drafts](https://www.notion.so/astronomerio/Internal-Drafts-e09f6e1e64464f1cb4b3468a549bb45f).

```

> 3. Copy the template into your new page 

Reviewer: ```suggestion
3. Copy and paste the completed template into your new page.

4. Notify the docs team when your content is ready for review.

```

## What is the name of your feature? If it doesn't have a name, what's the tagline?

## What is the purpose of the feature? 

> ## What problems does it solve? What new possibilities does it enable for customers.

Reviewer: ```suggestion
## What problems does it solve? What will it allow customers to accomplish?

```

> ## Who is the feature for? For example, “An advanced data practitioner who wants to view code changes between DAG versions”

Reviewer: ```suggestion
## Who is the intended audience? For example, “An advanced data practitioner who wants to view code changes between DAG versions”

```

> ## If applicable, write a numbered procedure for how a user turns on or uses the feature. Don’t worry about style or layout. Try and capture each action that the user must complete to implement the feature successfully. 

Reviewer: ```suggestion
## If applicable, write a numbered procedure for how a user configures or implements the feature. Don’t worry about style or layout. Try and capture each action that is necessary for success. 

```

# Write your own new feature documentation

To have the docs team create documentation for your new feature: 

1. Complete this template.

2. Add a new page to [Internal drafts](https://www.notion.so/astronomerio/Internal-Drafts-e09f6e1e64464f1cb4b3468a549bb45f).

3. Copy and paste the completed template into your new page.

4. Notify the docs team when your content is ready for review.

> 

Reviewer 2: I really like this template - we have a similarly titled resource (https://www.notion.so/astronomerio/Submit-a-Docs-PR-a3ca9018fdbc4206b120a6d530a8401b?pvs=4) but this notion doc is more about making PR's. Is there any reason we need the PR instructions in a separate place? Would they fit correctly in the Readme?
